javaparser-core/src/main/java/com/github/javaparser/ast/visitor/VoidVisitor.java

```diff
@@ -41,189 +35,180 @@
  */
 public interface VoidVisitor<A> {
 
-    //- Compilation Unit ----------------------------------
-
-    void visit(CompilationUnit n, A arg);
+    void visit(AnnotationDeclaration n, A arg);
 
-    void visit(PackageDeclaration n, A arg);
+    void visit(AnnotationMemberDeclaration n, A arg);
```

Comment:
I would consider adding some meta-data that indicates this grouping first - but it's not a priority since I want this to actually generate the visitors first :)